.vscode/settings.json

```diff
@@ -4,4 +4,5 @@
         "rust-analyzer"
     ],
     "rust-analyzer.check.command": "clippy",
+    "rust-analyzer.showUnlinkedFileNotification": false,
```

Comment:
this is annoying and not very useful. I would like to leave it in but if we don't want to I'll remove it.

Reply:
Idk what this does